Atomic/backends/_docker.py

```diff
@@ -328,7 +329,8 @@ def pull_image(self, image, remote_image_obj, **kwargs):
         local_image = self.has_image(image)
         if local_image is not None:
             if self.already_has_image(local_image, remote_image_obj):
-                raise ValueError("Latest version of {} already present.".format(image))
+                util.write_err("Latest version of {} already present.".format(image))
```

Comment:
Maybe I'm misunderstanding the desired end game here, but I thought you'd just replace the old 'raise ValueError' call here with a 'return 0' then let the self_inspect fire in the update call as a double (triple) check.